examples/glue_benchmark/glue.py

```diff
@@ -112,200 +69,127 @@ def split_features(x):
-
-    if FLAGS.load_finetuning_model:
-        with strategy.scope():
-            finetuning_model = tf.keras.models.load_model(
-                FLAGS.load_finetuning_model
-            )
-    else:
-        with strategy.scope():
-            loss = keras.losses.SparseCategoricalCrossentropy(from_logits=True)
-            metrics = [keras.metrics.SparseCategoricalAccuracy()]
-            if FLAGS.task_name == "stsb":
-                num_classes = 1
-                loss = keras.losses.MeanSquaredError()
-                metrics = [keras.metrics.MeanSquaredError()]
-            elif FLAGS.task_name in (
-                "mnli",
-                "mnli_mismatched",
-                "mnli_matched",
-                "ax",
-            ):
-                num_classes = 3
-            else:
-                num_classes = 2
-
-            # ----- Custom code block starts -----
-            # Users should change this `BertClassifier` to your own classifier.
-            # Commonly the classifier is simply your model + several dense layers,
-            # please refer to "Make the Finetuning Model" section in README for
-            # detailed instructions.
-            bert_model = keras_nlp.models.BertBackbone.from_preset(
-                "bert_base_en_uncased"
-            )
-            finetuning_model = keras_nlp.models.BertClassifier(
-                backbone=bert_model,
-                num_classes=num_classes,
-            )
-            # ----- Custom code block ends -----
-            lr = tf.keras.optimizers.schedules.PolynomialDecay(
-                FLAGS.learning_rate,
-                decay_steps=train_ds.cardinality() * FLAGS.epochs,
-                end_learning_rate=0.0,
-            )
-            optimizer = tf.keras.optimizers.experimental.AdamW(
-                lr, weight_decay=0.01, global_clipnorm=1.0
-            )
-            optimizer.exclude_from_weight_decay(
-                var_names=["LayerNorm", "layer_norm", "bias"]
-            )
-            finetuning_model.compile(
-                optimizer=optimizer,
-                loss=loss,
-                metrics=metrics,
-            )
-
-        finetuning_model.fit(
-            train_ds,
-            validation_data=val_ds,
-            epochs=FLAGS.epochs,
-        )
-    with strategy.scope():
-        if FLAGS.submission_directory:
-            generate_submission_files(finetuning_model, test_ds, idx_order)
-    if FLAGS.save_finetuning_model:
-        # Don't need to save the optimizer.
-        finetuning_model.optimizer = None
-        finetuning_model.save(FLAGS.save_finetuning_model)
+    # Start training
+    print("Starting Training...")
+
+    st = time.time()
+    model.fit(train_ds, validation_data=validation_ds, epochs=FLAGS.epochs)
+    et = time.time()
+
+    print("Training Finished!")
+    print(
+        f"Training took :: {(et-st):.4f} seconds, or {((et-st)/60):.2f} minutes, or {((et-st)/3600):.2f} hours!"
```

Comment:
We can just keep the seconds output, and let's call it `wall_time` which is the official name.

Reply:
We should also report the validation accuracy and `examples_per_second`, which is `epochs * number_of_records / wall_time`.